Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrieveAuditTransformer_g0.java

```diff
@@ -26,18 +26,19 @@
  */
```

Comment:
I checked this class as well and it appears to not be used. You're pulling the request/response from the orchestratable in the class that's supposed to call this.